sites/partners/src/pages/listings/[id]/applications/index.tsx

```diff
@@ -189,14 +201,39 @@ const ApplicationsList = () => {
                       variant="primary-outlined"
                       size="sm"
                       className="mx-1"
-                      onClick={() => onExport()}
+                      onClick={() => setIsTermsOpen(true)}
                       loadingMessage={csvExportLoading && t("t.formSubmitted")}
                     >
                       {t("t.export")}
                     </Button>
                   </div>
                 }
               />
+              <ExportTermsDialog
+                dialogHeader={t("applications.export.dialogHeader")}
+                isOpen={isTermsOpen}
+                onClose={() => setIsTermsOpen(false)}
+                onSubmit={onSubmit}
+              >
+                <p>{t("applications.export.dialogSubheader")}</p>
+                <h2 className={styles["terms-of-use-text"]}>
+                  {t("applications.export.termsOfUse")}
+                </h2>
+                <span>
+                  {t("applications.export.termsBodyOne")}
+                  <a href="https://mtc.ca.gov/doorway-housing-portal-terms-use">
+                    {t("applications.export.termsOfUse")}
+                  </a>
+                  {t("applications.export.termsBodyTwo")}
+                  <a href="https://docs.google.com/document/d/1W4tIMtUMwz4KqdcO5f4yZi0R5AU74P3B/edit?usp=sharing&ouid=105961542504967611474&rtpof=true&sd=true">
+                    {t("applications.export.doorwayPartnersManual")}
```

Comment:
We shouldn't be concatenating strings for translations, could we include the URL in the string and pass in the string and URL as vars?

Reply:
Per oflfine conversation, I've moved these strings to the partners json files @emilyjablonski